yml/imsim.yaml

```diff
@@ -7986,157 +8036,165 @@ tables:
     "@id": "#DiaSource.isDipole"
     datatype: boolean
     mysql:datatype: BOOLEAN
-    description: Flag indicating diaSource is classified as a dipole
+    description: Flag indicating diaSource is classified as a dipole.
     fits:tunit:
+  - name: time_processed
+    "@id": "#DiaSource.time_processed"
+    datatype: timestamp
+    length: 6
+    nullable: false
+    description: Time when this record was generated.
```

Comment:
This might be a little terse, maybe people would get a better flavor with something like "Time when the image was processed and this DiaSource record was generated"?